src/main/java/io/github/moulberry/notenoughupdates/profileviewer/rift/RiftPage.java

```diff
@@ -95,6 +96,8 @@ public void drawPage(int mouseX, int mouseY, float partialTicks) {
 
 		JsonObject riftData = profileInfo.getAsJsonObject("rift");
 		JsonObject riftInventory = riftData.getAsJsonObject("inventory");
+		// TODO change everything to RiftJson walker moment
+		RiftJson rift = new GsonBuilder().create().fromJson(riftData, RiftJson.class);
```

Comment:
Parsing JSON like this is relatively expensive, and will throw on an invalid payload.